builder/image/Dockerfile

```diff
@@ -13,6 +13,10 @@ RUN curl -sSL -o /usr/local/bin/etcdctl https://s3-us-west-2.amazonaws.com/opdem
 RUN curl -sSL -o /usr/local/bin/confd https://s3-us-west-2.amazonaws.com/opdemand/confd-v0.5.0-json \
     && chmod +x /usr/local/bin/confd
 
+# install herokuish
+RUN curl -sSL https://github.com/aledbf/herokuish/releases/download/v0.1.2/herokuish_0.1.2_linux_x86_64.tgz \
```

Comment:
@johanneswuerbach I want that too but in the current implementation he variable (slug_path)[https://github.com/gliderlabs/herokuish/issues/5] is a constant so that is a problem in case of concurrent builds